### PR TITLE
Separate CSP into its own file

### PR DESCRIPTION
**Pull request by lukaw3d**
Related to https://github.com/oasisprotocol/oasis-wallet-web/issues/204
Fixes https://github.com/oasisprotocol/oasis-wallet-web/pull/518#discussion_r762164937

### CODE REVIEW

File: internals/getCsp.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      'self'
+      'report-sample';
+    style-src
+      'self'
+      fonts.googleapis.com
+      'unsafe-inline'
+      'report-sample';
+    font-src
+      'self'
+      fonts.gstatic.com;
```
Comment (matevz) [inline internals/getCsp.js:13]: Can you please look at https://github.com/oasisprotocol/oasis-wallet-web/issues/526
Eventually we should move away from connecting to Google services.

File: internals/getCsp.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const csp = `
+    default-src 'none';
+    script-src
+      'self'
+      'report-sample';
+    style-src
+      'self'
+      fonts.googleapis.com
```
Comment (matevz) [inline internals/getCsp.js:8]: This should go away once https://github.com/oasisprotocol/oasis-wallet-web/issues/526 is implemented.

**Review by matevz — APPROVED** [review]
I would prefer implementing https://github.com/oasisprotocol/oasis-wallet-web/issues/526 first so we don't depend on google services anymore.

File: internals/getCsp.js (hunk 1 above)
Comment (lukaw3d) [reply internals/getCsp.js:19]: +1

Added this and other future CSP improvements from https://github.com/oasisprotocol/oasis-wallet-web/pull/236 as TODO comment

File: README.md [hunk 3]
```diff
@@ -107,7 +107,8 @@ docker-compose up -d
 yarn start # Run this in another terminal to keep it open
 yarn cypress:run
 
-# Manually check that content-security-policy in ./internals/serve-with-csp.json doesn't break any functionality
+# Manually check that content-security-policy in ./internals/getCsp.js doesn't break any functionality
+yarn --silent print-csp
```
Comment (lukaw3d) [inline README.md:111]: @tjanez new command. Alternatively you can copy https://github.com/oasisprotocol/oasis-wallet-web/blob/3291206233849475f45aaed1d26d319a1d8f0895/internals/getCsp.js#L6-L33

**Pull request merged**